File: ws/tipantiza/u2/ws16Libraries/StoreApp/src/ec/edu/espe/storeapp/view/Store.java

package ec.edu.espe.storeapp.view;

import ec.edu.espe.storeapp.model.Product;
import utils.Tax;

/**
 *
 * @author Alexander Tipantiza < The_Softwarrios of Espeyour>
 */
public class Store {
    public static void main(String[] args) {
    int id;
    String description;
    float price;
    float pvp;
    
    Product product;
    
    id = 1;
    description = "computer";
    price = 100;
    
    pvp = Tax.computeTotal(price, 15F);
    product = new Product(id, description, price);
        System.out.println("product--->" + product);
        
    Product product2;
    id = 2;
    description = "mouse";
    price = 1000;
    
    
    pvp = Tax.computeTotal(price, 15F);
    product2 = new Product(id, description, price);
        System.out.println("product--->" + product2);
            
    
    
    
    
    }

}

File: hw/sanchez/u2/hw09Libraries/StoreAppJS/view/Store.js

//author Joel Sanchez <The_Softwarriors at ESPE>

const Product = require('../model/Product');
const Tax = require('taxeslib');

class Store {
    static main() {

        console.log("         STORE APP - WITH TAXESLIB");

        
        // Product 1
        let id = 1;
        let description = "computer";
        let price = 100.0;
        
        let product = new Product(id, description, price);
        console.log("product --->" + product.toString());
        
        // Product 2
        id = 2;
        description = "mouse";
        price = 1000.0;
        
        let product2 = new Product(id, description, price);
        console.log("product 2 ---> " + product2.toString());
        
        // Product 3 (with custom PVP)
        id = 3;
        description = "keyboard";
        price = 50.0;
        let customPvp = 75.0;
        
        let product3 = new Product(id, description, price, customPvp);
        console.log("\nproduct 3 ---> " + product3.toString());
        
    }
}

if (require.main === module) {
    Store.main();
}

module.exports = Store;